src/dsp/pdr.c

```diff
@@ -115,6 +115,114 @@ int pldm_pdr_add_check(pldm_pdr *repo, const uint8_t *data, uint32_t size,
 	return 0;
 }
 
+static inline uint32_t get_new_record_handle(const pldm_pdr *repo)
```

Comment:
this function is removed and combined in pldm_pdr_add_check() API itself. Can you please remove this API?

Reply:
done